src/components/ArticleContainer.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState, useContext } from "react";
+import { Route, Routes, Link } from "react-router-dom";
+import axios from "axios";
+import { getAllArticles } from "../utils";
+
+const ArticleContainer = ({}) => {
+    const [allArticles, setAllArticles] = useState([]);
+
+    useEffect(() => {
+      getAllArticles()
+        .then(({ articles }) => {
+          console.log(articles);
+          setAllArticles(articles);
+        })
+        .catch((error) => {
+          console.log("error");
+        });
+    }, []);
+  
+    return (
+      <>
```

Comment:
This fragment is not a semantic element, consider section, article, something that tells any screen reader that it is important.